pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -276,9 +276,10 @@ def __init__(  # pylint: disable=too-many-arguments
                     f"The {kernel_name} is not supported and currently "
                     "only 'Local' and 'NonZeroRandom' kernels are supported."
                 )
-            shots = shots if isinstance(shots, Sequence) else [shots]
-            if any(num_burnin >= s for s in shots):
-                raise ValueError("Shots should be greater than num_burnin.")
+            if shots is not None:
+                shots_list = shots if isinstance(shots, Sequence) else [shots]
+                if any(num_burnin >= s for s in shots_list):
+                    raise ValueError("Shots should be greater than num_burnin.")
```

Comment:
This is a temporary walk around until https://github.com/PennyLaneAI/pennylane-lightning/pull/1222 is merged